docs/superpowers/plans/2026-04-09-go-backend-tests.md

```
@@ -0,0 +1,1220 @@
+- Create: `internal/ee/audit/*_test.go` — audit log capture
+
+First, audit the actual EE package structure:
+
+```bash
+find internal/ee -type f -name '*.go' | grep -v _test | sort
+```
+
+Then write tests following the same mock + testify pattern. Key cases per package:
+
+**`internal/ee/licensing/`**
+```go
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)